pkgs/applications/finance/tradingview/default.nix

```
@@ -0,0 +1,92 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, squashfsTools
+, makeWrapper
+, alsa-lib
+, atk
+, at-spi2-atk
+, cups
+, gtk3
+, libdrm
+, libsecret
+, libxkbcommon
+, mesa
+, pango
+, sqlite
+, systemd
+, wayland
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tradingview";
+  version = "2.6.1";
+  revision = "44";
+  src = fetchurl {
+    url = "https://api.snapcraft.io/api/v1/snaps/download/nJdITJ6ZJxdvfu8Ch7n5kH5P99ClzBYV_${revision}.snap";
+    sha512 = "1ddd344d68cfb24774403ce9392c100ae330db49d6e27d71c51913d6b03de69cdb5edc3b5c5c6b25d3169335836ee72ea24536a91d9be6da2f007020c3d134b5";
```

I converted this to a sri hash like this:

```
nix hash to-sri --type "sha512" 1ddd344d68cfb24774403ce9392c100ae330db49d6e27d71c51913d6b03de69cdb5edc3b5c5c6b25d3169335836ee72ea24536a91d9be6da2f007020c3d134b5 
sha512-Hd00TWjPskd0QDzpOSwQCuMw20nW4n1xxRkT1rA95pzbXtw7XFxrJdMWkzWDbucuokU2qR2b5tovAHAgw9E0tQ==
```

This should allow you to use the more idiomatic:
```suggestion
    hash = "sha512-Hd00TWjPskd0QDzpOSwQCuMw20nW4n1xxRkT1rA95pzbXtw7XFxrJdMWkzWDbucuokU2qR2b5tovAHAgw9E0tQ==";
```